<ui version="4.0" >
 <class>ImportAkenClass</class>
 <widget class="QDialog" name="ImportAkenClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Protokollitaja - Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="maximumSize" >
        <size>
         <width>210</width>
         <height>125</height>
        </size>
       </property>
       <property name="title" >
        <string>Eraldajad</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="checkBoxTab" >
          <property name="text" >
           <string>tab</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="0" column="1" >
         <widget class="QCheckBox" name="checkBoxKoma" >
          <property name="text" >
           <string>koma (,)</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="1" column="0" >
         <widget class="QCheckBox" name="checkBoxTuhik" >
          <property name="text" >
           <string>tühik</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QCheckBox" name="checkBoxSemik" >
          <property name="text" >
           <string>semikoolon (;)</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QCheckBox" name="checkBoxPunkt" >
          <property name="text" >
           <string>punkt (.)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>muu:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="muuEdit" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="2" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Vali leht millele importida:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sakiBox" />
         </item>
        </layout>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="okNupp" >
         <property name="text" >
          <string>Impordi</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="failistNupp" >
         <property name="text" >
          <string>Failist...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="maluNupp" >
         <property name="text" >
          <string>Vahemälust</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="loobuNupp" >
         <property name="text" >
          <string>Loobu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>778</width>
        <height>445</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>okNupp</sender>
   <signal>clicked()</signal>
   <receiver>ImportAkenClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>677</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>741</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loobuNupp</sender>
   <signal>clicked()</signal>
   <receiver>ImportAkenClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>699</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel" >
     <x>743</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
